utils/generic-components.stanza

```diff
@@ -105,12 +121,51 @@ public defn closest-std-val (v:Double, tol:Double) :
 
   match-val * pow(10.0, expon)
 
-; GENERICPASSIVES
+
+; ========================================================
+; ========== Generic passive components ==================
+; ========================================================
+; Generic ceramic capacitor from database (of 600k options)
+public defn ceramic-cap (params:Tuple<KeyValue>) :
+  val default-params = ["type" => "ceramic" 
+                        "mounting" => "smd" 
+                        "case" => get-valid-pkg-list()
+                        "minimum_quantity" => 1
+                        "min-rated-voltage" => MIN-CAP-VOLTAGE
+                        "min-stock" => 5 * DESIGN-QUANTITY]
```

Comment:
If you think that you are going to apply min-stock, min-quantity, mounting and case to most request, it maybe worth adding it to default in Capacitor(), Inductor(), Resistor(), or create an overloading function that has those defaults.

But sometimes, too many defaults kills readability, so looks great as it is also

Reply:
I'd like to get the parts solver considering the total number of instances of a part when making selections, and not just assume 5. Kinda fine for now because anything you realistically need 100 of per design is most likely in stock in large quantities anyway.